src/applications/personalization/dashboard-2/sass/dashboard.scss

```diff
@@ -1,3 +1,20 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
 [tabindex="-1"]:focus {
   outline: 0;
 }
+
+.dashboard {
+  .health-care {
+    hr {
+      background: $color-primary;
+      height: 3px;
+      width: 75%;
```

Comment:
Not to big of a fan of handling css this way, but it does the trick since we don't have a design utility class for anything less than auto or full width.